CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   removed in favour of the new `instrumentation` feature combined with
   [Grafana Tempo's span metrics](https://grafana.com/docs/tempo/latest/server_side_metrics/span_metrics/).
 
+### Added
+
+- Add new argument to the maker: `db-failsafe`. If enabled, the maker will not start if an error occurred when opening the database. This can come handy to prevent accidentally creating a new empty database in case database migration was unsuccessful.
```

Comment:
Alternative suggestion for the changelog entry (name got changed, and IMHO it's important to state that this behaviour is the default on the maker now. 

```suggestion
- Do not start the maker an error occurred when opening the database. This can come handy to prevent accidentally creating a new empty database in case database migration was unsuccessful. If needed, this behaviour can be bypassed by passing an additional argument to the maker: `ignore-migration-errors`.
```